java/src/org/certificatetransparency/ctlog/LogEntry.java

```diff
@@ -0,0 +1,12 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class LogEntry {
+  public Ct.X509ChainEntry x509Entry;
+  public Ct.PrecertChainEntry preCertEntry;
+
+  public LogEntry() {
```

Comment:
It was a punt on my behalf, but this confirms it:
http://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.8.9